interop-devnet/justfile

```diff
@@ -0,0 +1,37 @@
+
+devnet-setup:
+  bash create-chains.sh
+
+devnet-build-images:
+  PWD="$(pwd)" DOCKER_BUILDKIT=1 COMPOSE_DOCKER_CLI_BUILD=1 \
+    docker compose build --progress plain \
+       --build-arg GIT_COMMIT={git_commit} \
+       --build-arg GIT_DATE={git_date}
+
+devnet-up:
+  docker compose up -d l1 l1-bn l1-vc
+
+  docker compose up -d \
+    op-supervisor \
+    op-node-a op-batcher-a op-proposer-a \
+    op-node-b op-batcher-b op-proposer-b
+
+devnet-down:
+  # stops services, does not remove containers/networks
+  docker compose stop
+
+devnet-clean:
+  rm -rf ../.devnet-interop
+  # Stops services, and removes containers/networks
+  docker compose down
+  # Now manually clean up the related images and volumes
+  # Note: `justfile` interprets the curly brackets. So we escape them, by wrapping it with more, as a string, like Jinja2.
+  docker image ls 'interop-devnet*' --format='{{ '{{.Repository}}' }}' | xargs -r docker rmi
+  docker volume ls --filter name=interop-devnet --format='{{ '{{.Name}}' }}' | xargs -r docker volume rm
+
+devnet-logs:
+  docker compose logs -f
+
+supervisor-workaround:
+  cast rpc --rpc-url="http://localhost:9045" admin_addL2RPC "ws://l2-a:8546"
+  cast rpc --rpc-url="http://localhost:9045" admin_addL2RPC "ws://l2-b:8546"
```

Comment:
We don't need this workaround any more. Now that the Supervisor is accepting of not having an L2 connection for chains, the supervisor can be stood up, the RPC be determined, and then the URL be used in the OP Node start options.

Reply:
removed the workaround